crawlers/ten_ago_crawler.py

```
@@ -0,0 +1,138 @@
+
+def find_first_article_by_category(driver, urls, target_category):
+    for idx, url in enumerate(urls, 1):
+        print(f"\n🔎 ({idx}/{len(urls)}) [{target_category}] 기사 확인 중: {url}")
+        try:
+            driver.get(url)
+            time.sleep(1)
+
+            try:
+                tag = driver.find_element(By.CSS_SELECTOR, "a.category")
+                category = tag.get_attribute("title") or tag.text
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)